tailcall-wasm/README.md

```diff
@@ -0,0 +1,141 @@
+# Tailcall for Nodejs and Browser
+
+It is a simple library that allows you to execute graphql queries on Broswr and Nodejs.
+
+# Setup
+
+## Install (npm)
+
+```shell
+npm i @tailcallhq/tailcall-wasm
+```
+
+## Local setup
+
+````bash
+Go to tailcall-wasm dir
+
+```bash
+cd tailcall-wasm
+````
+
+The [package.json](package.json) file contains the scripts to run the project.
+
+## Nodejs
+
+For test build:
+
+```bash
+npm run "dev-node" # compiles in debug mode
+```
+
+For release build:
+
+```bash
+npm run "compile-node"  # compiles in release mode with optimizations
+```
+
+By default, the output is in the `node` folder.
+
+Now to run the example [index.js](example/node/index.js) file:
+
+```bash
+cd example/node && node index.js
+```
+
+## Browser
+
+For test build:
+
+```bash
+npm run "dev-browser" # compiles in debug mode
+```
+
+For release build:
+
+```bash
+npm run "compile-browser"  # compiles in release mode with optimizations
+```
+
+By default, the output is in the `browser` folder.
+
+Now to run the example [index.html](example/browser/index.html) file:
+
+Run simple http server in `tailcall-wasm` dir:
+
+```bash
+python3 -m http.server 8000
+```
+
+# Example
+
+### NodeJS
+
+```javascript
+const tc = require("@tailcallhq/tailcall-node")
+
+async function run() {
+  try {
+    let schema = "https://raw.githubusercontent.com/tailcallhq/tailcall/main/examples/jsonplaceholder.graphql"
+    let builder = new tc.TailcallBuilder()
```

Comment:
```js
let b = new tc.TailcallBuilder();

let exe = await b
  .withJsonConfig(" some json config>  ")
  .withYAMLConfig(" <some yml config>  ")
  .withConfigPath("https://raw.githubusercontent.com/tailcallhq/tailcall/main/examples/jsonplaceholder.graphql")
  .build();
```

Something like the above API would be better. 

We should not pass the content of the file and also the file path ever.

Reply:
The problem with that is, that we won't be able to access the code from ConfigReader, we can serialize Config but resolving that might be a problem.

To save memory, we can drop all the detected config files after building the Executor.